README.md

```diff
@@ -1,4 +1,35 @@
 # xfero
-File Transfer Orchestration software
+A practical File Transfer Orchestration solution for business
 
-This needs filling in
+## Why xfero?
+**THE MISSING LINK IN BUSINESS FILE TRANSFER SERVICES**
+
+Xfero is the complete orchestration framework for business file transfer services with a mission to simplify complex file transfer integration while giving business users the ability to manage configuration rather than relying on techies.
```

Comment:
Hi Steve, thanks for the comments. :+1: Do you intend the paragraph you shared to replace the one I had? And also, are you recommending that we use 'xfero' as opposed to 'Xfero' or 'XFERO'. If so I'd like to hear what @jaynebickerton thinks as our English and punctuation expert

Jayne, any thoughts... :smiley: